PUBLISHING.md

```diff
@@ -5,65 +5,86 @@ repository to [crates.io](https://crates.io/).
 
 **It is mostly intended for maintainers of the uefi-rs project.**
 
-## Overview
+## Goal: What We Want
+- We want precise entries in the [`CHANGELOG.md`] for every release that
+  **follow our existing format**. Changes are supposed to be added before a
+  release is performed.
+- We want git tags like `uefi-raw-v0.4.0` or `uefi-v0.25.0` for every release.
+- We want our crate dependencies published in the right order (if necessary):
+  - `uefi-services` depends on `uefi`
+  - `uefi` depends on `uefi-macros` and
+
+## How: Ways to Publish
+
+There are two major ways for releasing/publishing.
+
+### GitHub CI Workflow (**recommended**)
 
 1. Create a branch that updates the versions of all packages you want to
    release. This branch should include all related changes such as updating the
    versions in dependent crates and updating the changelog.
 2. Create a PR of that branch. The subject of the PR must start with `release:`,
-   the rest of the message is arbitrary.
-3. Once the PR is approved and merged, a Github Actions workflow will take care
+   the rest of the message is arbitrary. [Example](https://github.com/rust-osdev/uefi-rs/pull/1001)
+3. Once the PR is approved and merged, a GitHub Actions workflow will take care
    of creating git tags and publishing to crates.io.
 
-## Details of the release pull request
-
-For ensuring compatibility within the crates ecosystem,
-Cargo [recommends][cargo-semver] maintainers to follow the [semantic versioning][semver] guidelines.
+## Crates.io secret token
```

Comment:
Maybe change this to `####` so it's under Github CI Workflow, otherwise it breaks up the flow of "two major ways for releasing"

Reply:
yes, good catch. It was meant to be there.